DigitalLearningSolutions.Web/Views/TrackingSystem/Centre/Reports/_EditCourseCategoryFilters.cshtml

```diff
@@ -0,0 +1,80 @@
+﻿@using DigitalLearningSolutions.Data.Enums
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Centre.Reports
+@model EditFiltersViewModel
+
+@{
+  var courseSelected = Model.CustomisationId.HasValue;
+  var categorySelected = Model.CourseCategoryId.HasValue;
+}
+
+@if (Model.CanFilterCourseCategories) {
+  <div class="nhsuk-form-group">
+    <fieldset class="nhsuk-fieldset" aria-describedby="course-course-category-hint">
+      <div class="nhsuk-hint" id="course-course-category-hint">
+        Choose whether to filter based on course or course category
+      </div>
+
+      <div class="nhsuk-radios nhsuk-radios--conditional">
+        <div class="nhsuk-radios__item">
+          <input name="@nameof(Model.FilterType)"
+                 class="nhsuk-radios__input"
+                 id="course-filter-type-1"

```

Comment:
Did we decide to change these ID names and also make them a var to avoid typos or changes causing potential mismatch between that & the label?

Reply:
the id/for pair produce a warning if they are mismatched, so I'm leaving them. I'll change the aria-controls pairs.